cvmfs/util/posix.cc

```diff
@@ -1573,6 +1573,48 @@ bool SafeWriteToFile(const std::string &content,
   return retval;
 }
 
+void Panic() {
+#ifdef LIB_CVMFS_SERVER
+  throw "panic";
+#else
+  abort();
+#endif
+}
+
+// Most likely useless at this stage
+//
+// void Panic(const std::string msg) {
+//   LogCvmfs(kLogCvmfs, kLogDebug, "Panic: %s", msg);
+//
+//   #ifdef LIB_CVMFS_SERVER
+//     throw msg;
+//   #else
+//     abort();
+//   #endif
+// }
+//
+
+void Panic(const LogSource source, const int mask, const char *format, ...) {
```

Comment:
It would be useful if we could add the source code file and line where the panic was raised.

Reply:
I think for aborting, we don't need the LogSource and mask, and can thus drop the dependency on the logging module.